packages/jaeger-ui/src/components/App/Page.css

```diff
@@ -31,3 +31,12 @@ limitations under the License.
   right: 0;
   top: var(--nav-height);
 }
+
+.Page--content--embedded {
+  display: flex;
```

Comment:
The rules of` .Page--content--embedded` and .`Page--content`  are the same, except for   `top: var(--nav-height);`

We preserve the class  `.Page--content` , removing the last rule and create a new one , something like this 

```
.Page--content--no-embedded {
     top: var(--nav-height);
}
```

Then we can use those two classes in the component like this:

`cx({ 'Page--content': true, 'Page--content--no-embedded': !embedded });`

Whad do you think?

Reply:
fixed, @rubenvp8510 PTAL